docs/guide/quick-start.txt

```diff
@@ -0,0 +1,327 @@
+
+.. code-block:: js
+
+   const { MongoClient } = require('mongodb');
+   const assert = require('assert');
+
+   // Connection URL
+   const url = 'mongodb://localhost:27017';
+
+   // Database Name
+   const dbName = 'myproject';
+   const client = new MongoClient(url, { useNewUrlParser: true });
+
+   // Use connect method to connect to the server
+   client.connect().then(async function() {
+     console.log("Connected successfully to server");
+     const db = client.db(dbName);
+
+     try {
+       await insertDocuments(db);
+       await updateDocuments(db);
+     } finally {
+       await client.close();
+     }
+   });
+
+Remove a document
+-----------------
+
+Remove the document where the field **a** is equal to **3**.
+
+.. code-block:: js
+
+   async function removeDocument(db) {
+     // Get the documents collection
+     const collection = db.collection('documents');
+
+     // Delete document where a is 3
+     const result = await collection.deleteOne({ a : 3 });
+     assert.equal(1, result.result.n);
+     console.log("Removed the document with the field a equal to 3");
+
+     return result;
+   }
+
+Add the ``removeDocument`` method to your app:
+
+.. code-block:: js
+
+   const { MongoClient } = require('mongodb');
+   const assert = require('assert');
+
+   // Connection URL
+   const url = 'mongodb://localhost:27017';
+
+   // Database Name
+   const dbName = 'myproject';
+   const client = new MongoClient(url, { useNewUrlParser: true });
+
+   // Use connect method to connect to the server
+   client.connect().then(async function() {
+     console.log("Connected successfully to server");
+     const db = client.db(dbName);
+
+     try {
+       await insertDocuments(db);
+       await updateDocuments(db);
+       await removeDocuments(db);
+     } finally {
+       await client.close();
+     }
+   });
+
+Index a Collection
+------------------
+
+`Indexes <https://docs.mongodb.org/manual/indexes/>`_ can improve your application's
+performance. The following function creates an index on the **a** field in the
+**documents** collection.
+
+.. code-block:: js
+
+   async function indexCollection(db) {
+     const results = await db.collection('documents').createIndex({ a: 1 });
+     console.log(results);
+     return results;
+   }
+
+Add the ``indexCollection`` method to your app:
+
+.. code-block:: js
+
+   const { MongoClient } = require('mongodb');
+   const assert = require('assert');
+
+   // Connection URL
+   const url = 'mongodb://localhost:27017';
+
+   // Database Name
+   const dbName = 'myproject';
+   const client = new MongoClient(url, { useNewUrlParser: true });
+
+   // Use connect method to connect to the server
+   client.connect().then(async function() {
+     console.log("Connected successfully to server");
+     const db = client.db(dbName);
+
+     try {
+       await insertDocuments(db);
+       await indexCollection(db);
+     } finally {
+       await client.close();
+     }
+   });
```

Comment:
There used to be a link at the bottom here saying `For more detailed information, see the tutorials` with a  link to http://mongodb.github.io/node-mongodb-native/3.3/tutorials/main/. Since we no longer have that page, do we need an equivalent link here?